keyboards/dichotomy/keymaps/default/keymap.c

```diff
@@ -0,0 +1,518 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include "dichotomy.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```